sw/eventListener.js

```diff
@@ -116,6 +116,8 @@ const mergeNotification = (event, sw, payload, currentNotifications, tag, nid) =
   if (AMOUNT_TAGS.includes(compareTag)) {
     if (compareTag === 'REPLY') {
       title = `you have ${amount} new replies`
+    } else if (compareTag === 'THREAD') {
+      title = `you have ${amount} new follow-up replies`
```

Comment:
I was too slow to review / forgot to review after I requested a review from myself but I suspect we will want to merge this with `REPLY` later. I don't really need to know in my push notification at which level I received a reply. We also don't distinguish them in /notifications.

But we can keep it as-is for now and see. Just wanted to mention this as a UX thing to consider in the future :)